eZ/Publish/API/Repository/Tests/BaseTest.php

```diff
@@ -202,21 +202,21 @@ protected function getSetupFactory(): SetupFactory
      * @param mixed[] $expectedValues
      * @param \eZ\Publish\API\Repository\Values\ValueObject $actualObject
      */
-    protected function assertPropertiesCorrect(array $expectedValues, ValueObject $actualObject)
+    protected function assertPropertiesCorrect(array $expectedValues, ValueObject $actualObject, bool $strictComparison = false)
```

Comment:
-1, this should reflect PHPUnit API, at least on a method naming level (because there's this debt that we didn't design those assertions PHPUnit way, by extending proper classes).
So minimum: should be `assertPropertiesEqual`, `assertPropertiesSame`.

Reply:
Sorry but proper refactoring of  this method should be processes as separate dedicated story to don't obfuscate real sope of the JIRA issue and unblock people which waiting for the fix. Moved to https://jira.ez.no/browse/EZP-31534